Sources/environments/RpcCommunicator.swift

```diff
@@ -35,16 +35,33 @@ class UnityToExternalServicerImplementation : CommunicatorObjects_UnityToExterna
         return p.futureResult
     }
     
-    func initialize(request: CommunicatorObjects_UnityMessageProto, context: StatusOnlyCallContext) -> EventLoopFuture<CommunicatorObjects_UnityMessageProto> {
+    func initialize(request: CommunicatorObjects_UnityMessageProto, context: StatusOnlyCallContext) ->
+    EventLoopFuture<CommunicatorObjects_UnityMessageProto> {
         return handle(context, request)
     }
     
     func exchange(request: CommunicatorObjects_UnityMessageProto, context: StatusOnlyCallContext) -> EventLoopFuture<CommunicatorObjects_UnityMessageProto> {
+        print("========?>>>>>")
+     var rlIn = CommunicatorObjects_UnityRLInputProto()
```

Comment:
revert this part

Reply:
Indeed. I will clean it up as its still WIP :)

Reply:
or try this instead :)

```swift
    fileprivate func handle(_ context: StatusOnlyCallContext, _ request: CommunicatorObjects_UnityMessageProto) -> EventLoopFuture<CommunicatorObjects_UnityMessageProto> {
        var rlIn = CommunicatorObjects_UnityRLInputProto()
        rlIn.command = CommunicatorObjects_CommandProto.reset
        rlIn.sideChannel = request.unityOutput.rlOutput.sideChannel
        var response = CommunicatorObjects_UnityMessageProto()
        response.header.status = 200
        response.unityInput = CommunicatorObjects_UnityInputProto()
        response.unityInput.rlInput = rlIn
        response.unityInput.rlInitializationInput = CommunicatorObjects_UnityRLInitializationInputProto()
        response.unityInput.rlInitializationInput.capabilities.baseRlcapabilities = true
        response.unityInput.rlInitializationInput.packageVersion = "1.0.4"
        response.unityInput.rlInitializationInput.communicationVersion = "1.0.0"
        print("request")
        print(request)
        print("response")
        print(response)
        return context.eventLoop.makeSucceededFuture(response)
    }
    func initialize(request: CommunicatorObjects_UnityMessageProto, context: StatusOnlyCallContext) -> EventLoopFuture<CommunicatorObjects_UnityMessageProto> {
        return handle(context, request)
    }
    func exchange(request: CommunicatorObjects_UnityMessageProto, context: StatusOnlyCallContext) -> EventLoopFuture<CommunicatorObjects_UnityMessageProto> {
        return handle(context, request)
    }
```